### DISCUSSION TITLE
Another way of handling Task<Result<Maybe<T>>>?

### DISCUSSION THREAD

**Discussion by SLOPapa** [vkhorikov/CSharpFunctionalExtensions#539] [Q&A] [answered]
I'm probably doing something wrong but is there a way to reduce the angle brackets in such scenarios? I'm seeing methods that return a `Task<Result<Maybe<List<something>>>>` type of syntax. Of course, it works, but the syntax is cumbersome.

**Comment by vkhorikov** [comment, marked as answer]
Normally, the answer is to introduce strong typing, e.g in stead of `Dictionary<string, List<string>>` you may want to do `MyEmployeeRegistry` or something similar. So maybe wrap the `List<something>` into a record or a value object. It won't remove all the brackets, but it should reduce their number to a degree.
